etc/vzlogger_generic.schema.json

```diff
@@ -134,7 +134,9 @@
             { "$ref": "#/definitions/meter" },
             { "properties": {
                 "protocol": { "type": "string", "enum": ["s0"] },
-                "device": { "type": "string", "description": "device the meter is connected to. E.g. /dev/ttyUSB0" },
+                "device": { "type": "string", "description": "UART device the meter is connected to. E.g. /dev/ttyUSB0" },
+                "gpio": { "type": "integer", "default": -1, "description": "Number of GPIO port to be used. If this is set >-1 then device is ignored." },
```

Comment:
> Am 07.06.2015 um 07:50 schrieb andig <notifications@github.com>:
> 
> In etc/vzlogger_generic.schema.json <https://github.com/volkszaehler/vzlogger/pull/169#discussion_r31874160>:
> 
> > @@ -134,7 +134,9 @@
> >              { "$ref": "#/definitions/meter" },
> >              { "properties": {
> >                  "protocol": { "type": "string", "enum": ["s0"] },
> > -                "device": { "type": "string", "description": "device the meter is connected to. E.g. /dev/ttyUSB0" },
> > +                "device": { "type": "string", "description": "UART device the meter is connected to. E.g. /dev/ttyUSB0" },
> > +                "gpio": { "type": "integer", "default": -1, "description": "Number of GPIO port to be used. If this is set >-1 then device is ignored." },
> Write >= 0 rather than >-1?
> 
> 

Actually I’d be more interested whether the first GPIO is GPIO0 or GPIO1. I couldn’t find any info in the linux gpio doc. I used „>-1“ as default is -1.
> —
> Reply to this email directly or view it on GitHub <https://github.com/volkszaehler/vzlogger/pull/169/files#r31874160>.
> 

Gruß

Matthias